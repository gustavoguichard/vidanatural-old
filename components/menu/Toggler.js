import classnames from 'classnames'

export default ({ isOpen, onClick }) =>
  <a
    href="#menu"
    onClick={onClick}
    className={classnames('menu-icon', { 'is-open': isOpen })}
  >
    <div className="menu-icon-dash outer" />
    {isOpen || <div className="menu-icon-dash" />}
    <div className="menu-icon-dash outer" />
  </a>
