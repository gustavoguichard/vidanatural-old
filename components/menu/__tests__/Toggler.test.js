import { render } from 'react-testing-library'
import Toggler from '../Toggler'

it('renders correctly', () => {
  const { container } = render(<Toggler />)
  expect(container).toMatchSnapshot()
})

it('accepts props', () => {
  const { container } = render(<Toggler isOpen />)
  expect(container).toMatchSnapshot()
})
