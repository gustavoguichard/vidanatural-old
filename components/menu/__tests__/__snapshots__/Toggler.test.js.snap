// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`accepts props 1`] = `
<div>
  <a
    class="menu-icon is-open"
    href="#menu"
    title="Menu"
  >
    <div
      class="menu-icon-dash outer"
    />
    <div
      class="menu-icon-dash outer"
    />
  </a>
</div>
`;

exports[`renders correctly 1`] = `
<div>
  <a
    class="menu-icon"
    href="#menu"
    title="Menu"
  >
    <div
      class="menu-icon-dash outer"
    />
    <div
      class="menu-icon-dash"
    />
    <div
      class="menu-icon-dash outer"
    />
  </a>
</div>
`;
